src/lib/y2partitioner/widgets/configurable_blk_devices_table.rb

```diff
@@ -50,7 +50,10 @@ def opt
       # @macro seeAbstractWidget
       def init
         initial_sid = UIState.instance.row_sid
-        self.value = row_id(initial_sid) if initial_sid
+
+        # if we do not have valid sid, then pick first available device.
+        # Reason is to allow e.g. chain of delete like described in bsc#1076318
+        self.value = row_id(valid_sid?(initial_sid) ? initial_sid : devices.first)
```

Comment:
What happens when the last element is deleted? I guess we should do something like:

```
initial_sid =  UIState.instance.row_sid
initial_sid = devices.first.sid if !valid_sid?(initial_sid) && !devices.empty?
```
```

Reply:
well, it works, but it prints to log warning that widget with id "table:device:0". I will add that check.